src/Docker/Configuration/Container.php

```diff
@@ -24,6 +24,9 @@ public function getConfigTreeBuilder()
                     ->children()
                         ->booleanNode('safe')->defaultNull()->end()
                         ->scalarNode('image_tag')->defaultNull()->end()
+                        ->arrayNode('backend')->ignoreExtraKeys(true)->children()
```

Comment:
nevím jestli sem dávat extra keys, spíš bych řekl, že ne

Reply:
Daval jsem ho do vsech mist. kde jsem delal config builder. Pocitam s tim, ze v budoucnu do `backend` budou pribyvat nove volby.

Reply:
No to je v pořádku, že budou přibývat nové, ale tady je to rozhodující místo, kde jsou ty platné

Reply:
Jsi si jistej? Premyslel jsme nad tim tak, ze kdyz pridame nejakou novou option treba na strane nove Q, tak ji sem zacne posilat pres config a bez `ignoreExtraKeys` to tu zacne padat na nevalidnim configu, misto toho, aby se to jen silently preslo, protoze to dal zatim neni implementovane, ne?

Reply:
Na zaklade vysledku callu jsem teda dal pryc `ignoreExtraKeys` + jsem navic jeste config upravil tak, aby `backend` byl jako prazdne pole, i kdyz se vubec nenastavi.